common/data/data.json

```
@@ -798,7 +798,8 @@
   "findable_seasons": {
     "easter": [[3, 31], [4, 23]],
     "halloween": [[10, 20], [11, 5]],
-    "4july": [[6, 30], [7, 15]]
+    "4july": [[6, 30], [7, 15]],
+    "discordbirthday": [[5,6],[5,20]]
```

```suggestion
    "discord_birthday": [[5,6], [5,20]]
```

I changed it from `discordbirthday` to `discord_birthday` so you will have to make that change in other places below sorry

---

Done